API_IMPLEMENTATION.md

```diff
@@ -0,0 +1,241 @@
+# Forest API Implementation Report
+
+## Stats
+
+- Forest method count: 66
+- Lotus method count: 169
+- API coverage: 39.05%
+
+## Forest-only Methods
+
+These methods exist in Forest only and cannot be compared:
+
+- `Filecoin.AuthNew`
```

Comment:
I'm pretty confused. I thought that these all exist in Lotus.... I mean we implemented all of these so that we can be compatible with Lotus 🤔

Reply:
The only methods i can think of that would be in Forest and not in Lotus are methods prefixed with `Forest.` instead of `Filecoin.`

Reply:
Looking into it, I think there's a small bug, in that when I compare the method names, I need to make sure the case is consistent. But stuff like AuthNew, it's just not there in the OpenRPC file, no auth methods are. And that's what this is really comparing. It's just comparing their auto-generated API schema against ours.

https://github.com/ChainSafe/forest/blob/8f51cea9464e809f0768670b4fe7cf2a8df5476f/node/rpc-api/static/full.json

Reply:
An example of the inconsistent casing:
`ChainGetTipsetByHeight`
vs
`ChainGetTipSetByHeight`

Actually, I'm not sure I'm going to fix that. That's not a bug in my script, that's an actual API incompatibility. I think proper casing should matter. And this is the kind of thing that reveals that discrepancy.